template/config.tpl/tyk/confs/api_candig.json.tpl

```diff
@@ -0,0 +1,157 @@
+{
+    "api_id": "21",
+    "jwt_issued_at_validation_skew": 0,
+    "jwt_expires_at_validation_skew": 0,
+    "upstream_certificates": {},
+    "use_keyless": false,
+    "enable_coprocess_auth": false,
+    "base_identity_provided_by": "",
+    
+    "proxy": {
+	"target_url": "${LOCAL_CANDIG_SERVER}",
+	"strip_listen_path": true,
+        "disable_strip_slash": false,
+	"listen_path": "/${TYK_LISTEN_PATH}",
```

Comment:
Not sure about the `/`. Is it supposed to be there?

Reply:
I think it was there before?

Reply:
```# listen_path is empty "", put a slash only if there is a path "/<path>"
export TYK_LISTEN_PATH=""```
I think we do not need that slash?